src/site/_includes/partials/post.njk

```diff
@@ -10,6 +10,10 @@
     } %}
   {% endif %}
 
+  <web-table-of-contents title="{{ title | md | safe }}" title-id="{{ title | md | safe | slugify }}">
```

Comment:
it looks like you pipe title through a number of filters both here and in h1.

to dry it up a bit you could use `set` maybe and set it as a variable?